src/Web/Controllers/AccountController.cs

```diff
@@ -29,19 +32,23 @@ public class AccountController : Controller
         private readonly IBasketService _basketService;
         private readonly IFraudProtectionService _fraudProtectionService;
         private readonly IHttpContextAccessor _contextAccessor;
+        private readonly IPasswordHasher<ApplicationUser> _passwordHasher;
 
         public AccountController(
             UserManager<ApplicationUser> userManager,
             SignInManager<ApplicationUser> signInManager,
             IBasketService basketService,
             IFraudProtectionService fraudProtectionService,
-            IHttpContextAccessor contextAccessor)
+            IHttpContextAccessor contextAccessor,
+            IPasswordHasher<ApplicationUser> passwordHasher)
```

Comment:
Do you need to inject this?  I would think you could just use it directly from the ```_userManager.PasswordHasher```?